### PR TITLE
[16.0][MIG] partner_default_delivery_carrier: Migration to 16.0

### PR DESCRIPTION
**Pull request by BT-mrobles**

### CODE REVIEW

File: partner_default_delivery_carrier/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Default Delivery Method",
+    "summary": "Allows defining default delivery methods for partners",
+    "version": "16.0.1.0.0",
```
Comment (BT-anieto) [inline partner_default_delivery_carrier/__manifest__.py:7]: This change should go on the migration commit.

**Review by BT-anieto — CHANGES_REQUESTED** [review]
Thank you for your Migration! You need to squash your commits as stated in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: partner_default_delivery_carrier/__manifest__.py (hunk 1 above)
Comment (BT-mrobles) [reply partner_default_delivery_carrier/__manifest__.py:7]: Done!

**Comment by BT-mrobles** [comment]
@BT-anieto Thank you for your feedback. Requested changes applied

**Pull request closed without merging**